lib/web/apiserver.go

```diff
@@ -1073,6 +1073,7 @@ func (h *Handler) bindDefaultEndpoints() {
 
 	// AWS IAM Roles Anywhere Integration Actions
 	h.GET("/webapi/scripts/integrations/configure/awsra-trust-anchor.sh", h.WithLimiter(h.awsRolesAnywhereConfigureTrustAnchor))
+	h.POST("/webapi/sites/:site/integrations/aws-ra/:name/validate", h.WithClusterAuth(h.validateAWSRolesAnywhereIntegration))
```

Comment:
Since we also validate generic AWS OIDC integration when they are created, should we have a generic validate integration name endpoint? 

https://github.com/gravitational/teleport/blob/d10e51c00071cbb013261c68e090d52a4bd52d51/lib/auth/integration/integrationv1/service.go#L250C2-L261C4

Reply:
The validation used is the same, but I think we should kept it in separate endpoints.
We have other non-AWS integrations and it wouldn't make sense to add it to them, or have the UI client known which ones must be validated to a specific rule.